content-repo/extra-docs/releases/20.8.0.md

```diff
@@ -0,0 +1,1090 @@
+
+
+---
+
+### New: Ivanti Heat Pack v1.0.0
+#### Integrations
+##### Ivanti Heat  
+Use the Ivanti Heat integration to manage issues and create Cortex XSOAR incidents from Ivanti Heat.
+#### Scripts
+##### IvantiHeatCloseIncidentExample  
+This is a sample script that demonstrates how to close an incident in Ivanti Heat. The script generates data of the closed incident in JSON format and writes it to the IvantiHeat.CloseIncidentJSON context path.
+##### IvantiHeatCreateIncidentExample  
+This is a sample script that demonstrates how to create an incident in Ivanti Heat. The script generates data of the created incident in JSON format and writes it to the IvantiHeat.CreateIncidentJSON context path.
+##### IvantiHeatCreateProblemExample  
+This is a sample script that demonstrates how to create a problem in Ivanti Heat. The script generates data of the created problem in JSON format and writes it to the IvantiHeat.CreateProblemJSON context path.
+
+
+---
+
+### New: Nozomi Networks Pack v1.0.0
+#### Integrations
+##### Nozomi Networks  
+The Nozomi Networks Guardian platform is a hardware or virtual appliance that is used to monitor OT/IoT/IT networks. It combines asset discovery, network visualization, vulnerability assessment, risk monitoring and threat detection in a single solution.
+  This integration is used to gather alerts and assets information from Nozomi.
+
+
+---
+
+### New: RecordedFuture v2 Pack v1.0.0
+#### Integrations
+##### Recorded Future v2  
+Unique threat intel technology that automatically serves up relevant insights in real time.
+#### Playbooks
+##### Recorded Future CVE Intelligence  
+CVE enrichment using Recorded Future intelligence.
+##### Recorded Future CVE Reputation  
+CVE reputation with Recorded Future SOAR enrichment.
+##### Recorded Future Domain Intelligence  
+Domain enrichment using Recorded Future intelligence
+##### Recorded Future Domain Reputation  
+Domain reputation using Recorded Future SOAR enrichment
+##### Recorded Future File Intelligence  
+File enrichment using Recorded Future intelligence
+##### Recorded Future File Reputation  
+File reputation using Recorded Future SOAR enrichment
+##### Recorded Future IOC Reputation  
+Entity Reputation using sub-playbooks
+##### Recorded Future IP Intelligence  
+IP Address Enrichment using Recorded Future Intelligence
+##### Recorded Future IP Reputation  
+IP address reputation using Recorded Future SOAR enrichment
+##### Recorded Future Threat Assessment  
+Threat Assessment using the Recorded Future SOAR Triage API and the context Phishing.
+##### Recorded Future URL Intelligence  
+URL Enrichment using Recorded Future intelligence
+##### Recorded Future URL Reputation  
+URL reputation using Recorded Future SOAR enrichment
+
+
+---
+
+### New: Sepio Pack v1.0.0
+#### IncidentTypes
+##### Sepio Systems  
+
+#### Integrations
+##### Sepio  
+Get Agent, Switches and Events from your Sepio Prime
+
+
+---
+
+### Archer Pack v1.0.0
```

Comment:
This pack is named `RSA Archer`. Not sure how we got this name. @yaakovi 
```suggestion
### RSA Archer Pack v1.0.0
```

Reply:
I saw it missing so I added it manually...